nixos/modules/services/x11/window-managers/i3.nix

```diff
@@ -23,6 +23,23 @@ in
           If left at the default value, $HOME/.i3/config will be used.
         '';
       };
+
+      debug = mkOption {
+        default = false;
+        example = true;
+        type = types.bool;
+        description = "Enable debug/verbose logging (see -V option)";
+      };
+
+      logFile = mkOption {
+        default = null;
+        example = "$HOME/.i3/i3log";
+        type = types.string;
```

Comment:
That's fine, except that if the user is not aware of this and doesn't configure it (as was the case with my automated build) then that evaluation error comes up.